apps/condo/domains/resident/tasks/helpers/sendBillingReceiptsAddedNotifications.js

```diff
@@ -1,226 +1,30 @@
-            const params = [
-                get(item, 'resident.address'),
-                get(item, 'accountNumber'),
-            ]
-
-            return makeAccountKey(...params)
-        })
-
-        for (const receipt of receipts) {
-            const params = [
-                get(receipt, 'property.address'),
-                get(receipt, 'account.number'),
-            ]
-            const receiptAccountKey = makeAccountKey(...params)
-            const consumers = consumersByAccountKey[receiptAccountKey]
-
-            // We have no ServiceConsumer records for this receipt
-            if (isEmpty(consumers)) continue
-
-            let successConsumerCount = 0
-
-            for (const consumer of consumers) {
-                const resident = get(consumer, 'resident')
-
-                // ServiceConsumer has no connection to Resident
-                if (!resident || resident.deletedAt) continue
-
-                // NOTE: (DOMA-4351) skip already notified user to get rid of duplicate notifications
-                if (notifiedUsers.has(resident.user.id)) continue
-
-                const success = await prepareAndSendNotification(context, receipt, resident)
-
-                if (success) notifiedUsers.add(resident.user.id)
-
-                successConsumerCount += success
-            }
-
-            if (successConsumerCount > 0) successCount += 1
-
-            lastReceipt = receipt
-        }
-
-        // Store receipt.createdAt as lastDt in order to continue from this point on next execution
-        if (isFunction(onLastDtChange) && !isEmpty(lastReceipt)) await onLastDtChange(lastReceipt.createdAt)
-
-        logger.info({ msg: `Processed ${skip} receipts of ${receiptsCount}.` })
-    }
-    logger.info({ msg: 'sent billing receipts', successCount, receiptsCount })
-}
-
-
-const sendBillingReceiptsAddedNotifications = async (resendFromDt = null) => {
-    const { prevMonthStart, thisMonthStart } = getStartDates()
-    const redisClient = getRedisClient()
-    const handleLastDtChange = async (createdAt) => { await redisClient.set(REDIS_LAST_DATE_KEY, createdAt) }
-
-    /**
-     * This represents min value for billingReceipt createdAt to start processing from
-     * 1. First of all use resendFromDt from CLI parameter if available, else
-     * 2. Use createdAt value from last success script execution stored in Redis, else
-     * 3. Use thisMonthStart
-     */
-    const lastDt = resendFromDt ? resendFromDt.replace(' ', 'T') : await redisClient.get(REDIS_LAST_DATE_KEY) || thisMonthStart
+    const BillingContexts = await find('BillingIntegrationOrganizationContext', {
+        updatedAt_gte: lastSendDate,
+        status: CONTEXT_FINISHED_STATUS,
+        deletedAt: null,
+    })
 
-    logger.info({ msg: 'stored date', storedDt: await redisClient.get(REDIS_LAST_DATE_KEY) })
+    logger.info({ msg: 'Billing contexts for pushes', data: BillingContexts })
```

Comment:
To keep variable name
```suggestion
    logger.info({ msg: 'Billing contexts for pushes', data: { BillingContexts } })
```